### ISSUE TITLE
A11y_.NetAspireDashboard_Resources_Resources_AI4D: Landmarks are not provided in "Resources" page.

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/aspire#2810]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier." 

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

### Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #Benchmark; #Win11; #ChromiumEdge; #FTP; #DesktopWeb; #.NET Core; #SH_.Net Aspire Dashboard_Web_Mar2024; #WCAG1.3.1; #Element:Landmarks;

### Environment Details:
Application Name: .Net Aspire Dashboard
URL: https://localhost:19888/
Chrome Version 122.0.2365.80 (Official build) (64-bit)
Window Version: Win 11
OS Build: 22621.3085

### Repro Steps:
1. Hit the URL:  https://localhost:19888/ and login with valid credentials.
2. TAB to "Resources" tab item and hit ENTER key.
3. Observe the issue.

### Actual Results
Landmarks are not provided in "Resources" page, such as Main landmark, navigation landmark.

### Expected Results:
Landmarks should be provided in "Resources" page, such as Main landmark, navigation landmark.

### User Impact:
Screen reader users will not get information regarding structure of the page.

### Attachment:

![2810_A11y_ NetAspireDashboard_Resources_Resources_AI4D](https://github.com/dotnet/aspire/assets/90901519/a11da01b-2088-49fb-a131-53e53bd82f52)

**Comment by leslierichardson95** [comment]
Are you currently using the latest version of Aspire?  The screenshot you shared is a version of the dashboard from an earlier version.

**Comment by adamint** [comment]
This is not a current version of the aspire dashboard. I suspect this is coming from dotnet/eShop not using a bleeding edge version of the aspire packages.

Let’s retest using the playground/TestShop sample app within this repository. @PoojaNamde

**Comment by PoojaNamde** [comment]
#Closed; 

Closing this bug as per above comment.
